lib/traject/config/sirsi_config.rb

```diff
@@ -1918,7 +1918,13 @@ def marc_008_date(byte6values, byte_range, u_replacement)
 #    vernacular: [['The same, but pulled from the matched vernacular fields']]
 #    unmatched_vernacular: [['The same, but pulled from any unmatched vernacular fields']]
 to_field 'toc_struct' do |marc, accumulator|
-  formatted_chapter_regex = Regexp.union(/[^\S]--[^\S]/, /      /, /(?=(?:Chapter|Section|Appendix|Part|v\.) \d+[:\.-]?\s+)/i,  /(?=(?<!Chapter|Section|Appendix|Part|v\.) \d+[:\.-]?\s+)/i, /(?=(?:Appendix|Section|Chapter) [XVI]+[\.-]?)/i)
+  formatted_chapter_regex = Regexp.union(
+    /[^\S]--[^\S]/,
+    /      /,
+    /(?=(?:Chapter|Section|Appendix|Part|v\.) \d+[:\.-]\s+)/i,
```

Comment:
I wonder if this was copy-pasta and should have been:

```
/(?=(?:Chapter|Section|Appendix|Part|v\.) \d+[:\.-]?\s+)/i,
/(?=(?<!Chapter|Section|Appendix|Part|v\.) \d+[:\.-]\s+)/i
```

(Punctuation required if it isn't preceded by one of those words). I'm 👍 to shipping this now, but I wonder if we should grab a ton of real-world data to throw at it too?